main.go

```diff
@@ -24,6 +24,9 @@ import (
 	"strings"
 	"time"
 
+	// load credential helpers
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
Would it be better to put the comment on the end of the line to avoid it parting company from the line and/or getting deleted by goimports and such like? I don't know what the best practice is, so this is just a thought.

Reply:
This is the common practice. The comment applies to the group of imports not the individual import, it just happens that we only have one right now. The comment will be preserved by tools at the top of the import group.